lib/rucio/daemons/replicarecoverer/suspicious_replica_recoverer.py

```diff
@@ -80,13 +73,32 @@ def declare_suspicious_replicas_bad(once: bool = False, younger_than: int = 3, n
     :returns: None
     """
 
+    run_daemon(
+        once=once,
+        graceful_stop=GRACEFUL_STOP,
+        executable=argv[0],
+        logger_prefix='replica_recoverer: ',
```

Comment:
I would suggest dropping the `: ` suffix. Refer to the format in https://github.com/rucio/rucio/blob/771a80183a60aff917031d8b182f574d35f325e1/lib/rucio/daemons/common.py#L81